src/shared/utils/cookieHelper.js

```diff
@@ -20,6 +20,11 @@ export const setUserAuthCookie = ({ token, user }) => {
   cookies.set('verified', user.verified, cookieOptions);
 };
 
+export const setVerified = (verified) => {
```

Comment:
I'd like to see the function name be `setUserVerifiedCookie` and the argument be `isVerified` to indicate boolean usage.